ops/env.conf

```diff
@@ -1,9 +1,12 @@
 env TAG;
 env DB_ADAPTER;
+env MANAGEMENT_TAG
```

Comment:
good catch - this is needed so that in production these variables get through to the running passenger processes. Otherwise they will always appear blank to the application causing this code to seem like it doesn't work (when it does but isn't getting the data it wants)